master_server.py

```diff
@@ -29,13 +29,15 @@
 
 from other.utils import create_server_from_base
 
+from mh.state import Cache
 
-def create_servers(silent=False, debug_mode=False):
+def create_servers(server_id, silent=False, debug_mode=False):
     """Create servers and check if it has ui."""
     servers = []
     has_ui = False
-    for module in (OPN, LMP, FMP, RFP):
+    for module in ((OPN, LMP, FMP, RFP) if server_id==0 else (FMP,)):
         server, has_window = create_server_from_base(*module.BASE,
+                                                     server_id=server_id,
```

Comment:
This is hackish and most likely have better places where they could be integrated (a proper config file, might be needed).